billing/catalog.go

```diff
@@ -0,0 +1,77 @@
+package billing
+
+import (
+	"context"
+	"strings"
+
+	billingpb "google.golang.org/genproto/googleapis/cloud/billing/v1"
+)
+
+type computeEngineCatalog struct {
+	service       string
+	coreInstances map[string][]*billingpb.Sku
+	RAMInstances  map[string][]*billingpb.Sku
+}
+
+func newComputeEngineCatalog() *computeEngineCatalog {
+	c := new(computeEngineCatalog)
+	c.service = "services/6F81-5844-456A"
+	c.coreInstances = map[string][]*billingpb.Sku{}
+	c.RAMInstances = map[string][]*billingpb.Sku{}
+	return c
+}
+
+var computeEngineCatalogPtr *computeEngineCatalog
```

Comment:
Could computeEngineCatalogPtr be initialized here?

```

var computeEngineCatalogPtr *computeEngineCatalog = newComputeEngineCatalog()
```

Then the lazy initialization in the two `Get*SKUs` functions should not be needed.

Reply:
Sure, will do